patient/README.md

```diff
@@ -0,0 +1,75 @@
+# Patient Mapping
+
+## JSON FHIR > XML HL7
+
+A Patient resource is mapped from a HL7 Patient.
+
+| Mapped to (JSON FHIR Immunization field)        | Mapped from (XML HL7 / other source)                                                              |
+|-------------------------------------------------|---------------------------------------------------------------------------------------------------|
+| id                                              | set value = Generated GUID                                                                        |

```

Comment:
The adaptor is generating a UUID here, which isn't converted to uppercase . A GUID would be output as uppercase, see [here](https://stackoverflow.com/questions/246930/is-there-any-difference-between-a-guid-and-a-uuid). However, I've said UUID elsewhere in the docs, where the adaptor does an uppercase conversion.

Reply:
Doesn't technically matter as FHIR allows uppercase or lowercase [IDs](https://build.fhir.org/datatypes.html#id) .